FASTN.ftd

```diff
@@ -933,7 +933,8 @@ skip: true
   document: blog/external-css.ftd
 - Using Custom Breakpoints: /breakpoint/
   document: blog/breakpoint.ftd
- 
+- Understanding Loops: /loops/
+  document: blog/loops.ftd
```

Comment:
Change the url to `/loop/`, so it matches with our `/record/`, `/list/` etc URL. Else we will have to think if it is `/loop/` or `/loops/`. Also add a redirect entry from `/loops/ -> /loop/`.